fedmsg_meta_fedora_infrastructure/tests/fmn.py

```diff
@@ -243,6 +243,35 @@ class TestConfirmationUpdateStatus(Base):
         }
     }
 
+class TestRuleUpdate(Base):
+    """ Messages of this type are published whenever **the status of a
+    confirmation changes** in the `Fedora Notifications
```

Comment:
I'm not sure what you mean by ``the status of a confirmation changes`` here, maybe we could rephrase this a bit